packages/execution-environments/src/common/endowments/crypto.ts

```diff
@@ -0,0 +1,20 @@
+import { rootRealmGlobal } from '../globalObject';
+
+const createCrypto = () => {
+  if ('crypto' in rootRealmGlobal && 'SubtleCrypto' in rootRealmGlobal) {
+    return {
+      crypto: rootRealmGlobal.crypto,
+      SubtleCrypto: rootRealmGlobal.SubtleCrypto,
+    };
+  }
+  // @todo Figure out if this is enough
+  /* eslint-disable-next-line @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires, node/global-require */
+  const crypto = require('crypto').webcrypto;
```

Comment:
We should add a note that this API is experimental: https://nodejs.org/api/webcrypto.html#web-crypto-api